Packages/vcs/Lib/vcs2vtk.py

```diff
@@ -1199,7 +1199,7 @@ def prepFillarea(renWin, farea, cmap=None):
             if opacity is not None:
                 opacity = farea.opacity[i]
         else:
-            opacity = 100
+            opacity = None
```

Comment:
If it was 100 it would not respect the color's own opacity. so if you set to (100,0,0,50) you wolud have a filled red square instead of half transparent square.